packages/hoppscotch-ui/src/assets/scss/styles.scss

```diff
@@ -137,10 +137,10 @@ a {
 
 .cm-tooltip {
   .tippy-box {
-    @apply shadow-none;
+    @apply shadow-none #{!important};
     @apply fixed;
     @apply inline-flex;
-    @apply -mt-8;
+    @apply -mt-7.5;
```

Comment:
Margin of 7.5 isn't supported in Tailwind and we are moving to tailwind in the next release. Hence this style might not be applied.